workflows/mark05.smk

```diff
@@ -0,0 +1,133 @@
+"""Experiment mark05
```

Comment:
This is very nice and easy to follow! Great progress on writing Snakemake workflows :slightly_smiling_face: